agent/src/iconfucius/client/README.md

```
@@ -0,0 +1,69 @@
+# 2. Start the proxy server (in one terminal)
+npm run proxy
+
+# 3. Start the dev server (in another terminal)
+npm run dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) in your browser.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)